src/fpylll/util.pyx

```diff
@@ -165,25 +165,25 @@ def compute_gaussian_heuristic(int block_size, double root_det, double gh_factor
     cdef FP_NR[double] gh_dist = 0.0
     cdef FP_NR[double] root_det_ = root_det
     cdef int gh_dist_expo = 0
-    compute_gaussian_heuristic_c[FP_NR[double]](gh_dist, gh_dist_expo, block_size, root_det_, gh_factor)
+    gaussian_heuristic_c[FP_NR[double]](gh_dist, gh_dist_expo, block_size, root_det_, gh_factor)
```

Comment:
We should drop this function and turn it into a method on the GSO object to match fplll